### ISSUE TITLE
Open Modal Dialog in React on Load

### ISSUE THREAD

**Issue by DanielHabenicht** [telekom/scale#549]
Coming from #314 I tried as written there. 
But if I have an App like this and it wont open the modal. Also it won't open if I try to hook it up with an `onclick` event.

```tsx
function App() {
  const [isOpen, setIsOpen] = useState(false);

  useEffect(() => {
    setIsOpen(true);
  }, []);

  return (
    <ScaleAppShell
      style={{ width: '100%' }}
      logoHref={'/'}
      mainNavigation={[
        { name: 'Dashboard', id: 'search', onClick: navigate(ROUTES.dashboard) },
      ]}
    >
      <Switch>
        <Route path={ROUTES.dashboard}>
          <Dashboard />
        </Route>
      </Switch>
      <ScaleAppFooter
        variant={'minimal'}
        footerNavigation={[
          {
            name: 'Data privacy',
            id: 'Data privacy',
            href: 'link
            target: '_blank',
            icon: 'alert-imprint-dataprivacy',
          },
        ]}
      ></ScaleAppFooter>
      <ScaleModal id={'dataprivacymodal'} heading={'Data Privacy'} opened={isOpen}>
        <p>Modal Message</p>
      </ScaleModal>
    </ScaleAppShell>
  );
}

export default App;
```

**Comment by emMercado** [comment]
Hi! Could you try

const [isOpen, setIsOpen] = useState(false);

    /*     useEffect(() => {
            setIsOpen(true);
        }, []);
     */
            <p onClick={() => setIsOpen(true)}>Press of modal On</p>
            <ScaleModal  show={isOpen} onHide={() => setIsOpen(false)}>
                <p>Modal is ON</p>
                <img></img>
            </ScaleModal >

**Comment by DanielHabenicht** [comment]
That seems to work, but my goal is to open it on first load. 

I also already tried this or initializing the state with `true`
```tsx
  useEffect(() => {
    getUserData();
    setTimeout(() => {
      setIsOpen(true);
    }, 1000);
  }, []);
```

**Comment by nowseemee** [comment]
Many thanks for reporting this bug - the fix has been published: `3.0.0-beta.18`
